apps/public-docsite-v9/src/Concepts/Poster/Poster.tsx

```diff
@@ -0,0 +1,74 @@
+import * as React from 'react';
+import { FluentProvider, makeStyles, shorthands, tokens, webLightTheme } from '@fluentui/react-components';
+import { libraryInfo } from './metadata';
+import { AreaCard } from './AreaCard';
+import { AreaInfo } from './types';
+import { Legend } from './Legend';
+import { OverviewCard } from './OverviewCard';
+
+const useStyles = makeStyles({
+  root: {
+    ...shorthands.padding('40px'),
+    ...shorthands.overflow('scroll'),
```

Comment:
nit: set to `auto` instead to prevent double scrollbars from showing:
```suggestion
    ...shorthands.overflow('auto'),
```
![image](https://user-images.githubusercontent.com/8649804/169609057-f73de876-4eb7-4ad1-9d79-a0e56d2f225f.png)